sormas-api/src/main/java/de/symeda/sormas/api/externalmessage/processing/labmessage/AbstractLabMessageProcessingFlow.java

```diff
@@ -384,14 +439,17 @@ private CompletionStage<ProcessingResult<ExternalMessageProcessingResult>> handl
 		return relatedLabMessageHandler.handle(externalMessage).thenCompose(result -> {
 			AbstractRelatedLabMessageHandler.HandlerResultStatus status = result.getStatus();
 			if (status == AbstractRelatedLabMessageHandler.HandlerResultStatus.CANCELED) {
+				logger.debug("[MESSAGE PROCESSING] Canceled while handling as a related messages.");
```

Comment:
If only one message will be handled, please change the logging message to "...as a related message." here and at line 447.